modules/persistent-storage-csi-cloning-provisioning.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-cloning.adoc
+
+[id="persistent-storage-csi-cloning-provisioning_{context}"]
+= Provisioning a CSI volume clone
+
+Provisioning of a CSI volume clone is triggered by the creation of a cloned persistent volume claim (PVC) API object. The cloned PVC is pre-populated with the contents of another PVC. After creation, the cloned volume adheres to the same rules as any other persistent volume.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* A PVC created using a CSI driver that supports volume cloning.
+* Storage back end is configured for dynamic provisioning. Cloning support is not available for static provisioners.
+* Source and destination PVCs must exist in the same namespace.
+* Source and destination storage class must be the same.
+
+.Procedure
+
+Clones are provisioned like any other PVC, with the exception of adding a `dataSource` that references an existing PVC in the same namespace.
+
+To clone a PVC from an existing PVC:
+
+. Define the PersistentVolumeClaim object as shown in the following example:
+
++
+.pvc-clone.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-1-clone
+  namespace: mynamespace
+spec:
+  storageClassName: csi-cloning <1>
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+----
```

Comment:
The code snippet is just missing the most important part:
dataSource:
    kind: PersistentVolumeClaim
    name: pvc-1

Reply:
@fgiloux umm yes!! Thanks for pointing that out! Opened [PR 21010](https://github.com/openshift/openshift-docs/pull/21010) to add this in.